pcsx2/PAD/Linux/SDL/joystick.cpp

```diff
@@ -82,12 +82,23 @@ void JoystickInfo::Rumble(unsigned type, unsigned pad)
 		return;
 
 	if (m_haptic == nullptr)
-		return;
+	{
+		// If haptic is not implemented for a controller, try to use rumble API instead
+		static const Uint16 rumble_force = SDL_MAX_UINT16;
+		static const Uint16 big_motor[NB_EFFECT] = { 0, rumble_force, };
+		static const Uint16 small_motor[NB_EFFECT] = { rumble_force, 0, };
```

Comment:
Use uint16_t instead?

Reply:
I've followed the type used by `SDL_JoystickRumble()`. If you think `stdint.h` are better I'll change it

Reply:
Yeah change them for consistency.

Reply:
Done